### ISSUE TITLE
JOSS review: documentation

### ISSUE THREAD

**Issue by cescalara** [JohannesBuchner/BXA#19]
Hi, a few comments/questions on the documentation as part of the review:

- [x] Installation: Add the full list of dependencies to the install steps. I also needed `cython`, `tqdm`, `h5py` and `ultranest` as a minimal list to get things running. 
- [x] For both the sherpa/Xspec cases, it would be nice to see a full standalone demo including some example data. I know this is done in the scripts in `examples/` , but would be helpful if at least one case was treated notebook-style on the documentation pages.
- [x] Similarly for the `xagnfitter.py` script - you have a case with some example data in the Travis CI, it could be nice to also have this in the docs as a walkthrough for users.
- [x] The default ultranest output is cool, but could be distracting/annoying when running in certain settings. Could you show in the docs an easy interface in BXA to switch it off?
- [x] In the "convenience features" documentation
  - [x] Seems the example call of `gal.py` is a typo as refers to another script
  - [x] Would be nice to have some quick code showing exactly how to import chains in Xspec or run a script in parallel

**Comment by cescalara** [comment]
Linking [openjournals/joss-reviews/issues/3045](https://github.com/openjournals/joss-reviews/issues/3045)

**Comment by JohannesBuchner** [comment]
Hi,
>     * Installation: Add the full list of dependencies to the install steps. I also needed `cython`, `tqdm`, `h5py` and `ultranest` as a minimal list to get things running.
>     * In the "convenience features" documentation
>       
>       * Seems the example call of `gal.py` is a typo as refers to another script

>       * Would be nice to have some quick code showing exactly how to import chains in Xspec or run a script in parallel

>     * The default ultranest output is cool, but could be distracting/annoying when running in certain settings. Could you show in the docs an easy interface in BXA to switch it off?

I believe I fixed these now.

 
>     * For both the sherpa/Xspec cases, it would be nice to see a full standalone demo including some example data. I know this is done in the scripts in `examples/` , but would be helpful if at least one case was treated notebook-style on the documentation pages.
>     * Similarly for the `xagnfitter.py` script - you have a case with some example data in the Travis CI, it could be nice to also have this in the docs as a walkthrough for users.

I will see what I can do here, unfortunately the sherpa and Xspec codebases are complex and out of my control, and make inline plotting very difficult. Perhaps I can place README files in the example folders which show the steps and the expected output. Would that address your point?

**Comment by cescalara** [comment]
Great! Thanks, the updates look good. 

> I will see what I can do here, unfortunately the sherpa and Xspec codebases are complex and out of my control, and make inline plotting very difficult. Perhaps I can place README files in the example folders which show the steps and the expected output. Would that address your point?

Ah, I see what you mean. I think a README solution as you propose would be fine. The main point I wanted to make is that it would be nice for users to have a quick check of their install against expected output that they can run out of the box in a guided way.

**Comment by JohannesBuchner** [comment]
I added README files in the two folders:

https://github.com/JohannesBuchner/BXA/tree/master/examples/xspec
https://github.com/JohannesBuchner/BXA/tree/master/examples/sherpa

and the output of xagnfitter here:
https://johannesbuchner.github.io/BXA/xagnfitter.html

I believe this addresses the issue.

**Comment by cescalara** [comment]
Ok, I was thinking more some key plots (e.g. input spectrum, fit corner plot) with some explanation of what the example does or so for a quick visual check. Is there any reason you don't want to include the plots themselves? Text output is thorough but not so intuitive.

**Comment by JohannesBuchner** [comment]
It's a good idea, I will add it.

**Comment by JohannesBuchner** [comment]
Please have another look now and let me know if you have further suggestions.

**Comment by cescalara** [comment]
Looks great!
